about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+    <link rel="stylesheet" href="./about.css">
+    <title>About</title>
+  </head>
+  <body>
+  <header>
+    <nav class="nav-bar">
+        <i class="ham">
+        <i class="menuIcon fa fa-bars" ></i>
+        <i class="xIcon fa fa-close"></i>
+        </i>
+
+        <ul class="menu">
+          <li><a class="menuLink" href="index.html">Home</a></li>
+          <li><a class="menuLink" href="#">Program</a></li>
+          <li><a class="menuLink" href="about.html">About</a></li>
+          <li><a class="menuLink" href="#">Join</a></li>
+        </ul>
+
+        <ul class="topList">
+          <li><i class="fa fa-facebook-f"></i></li>
+          <li><i class="fa fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+      </ul>
+      </nav>
+      <nav class="desktopNavFirst">
+        <ul class="secondNavList">
+          <li class="secondNavItem"><a href="./about.html">About</a></li>
+          <li class="secondNavItem">Program</li>
+          <li class="secondNavItem">Join</li>
+          <li class="secondNavItem">Sponsor</li>
+          <li class="secondNavItem">News</li>
+          <li><button class="summitBtn" type="button">Summit</button></li>
+      </ul>
+       <a href="./index.html"> <img src="./images/images.png" alt="test"></a>   
+     </nav>
+  </header>
+    <section class="mainSection">
+        <div>
+          <h2>"Hello! Sharing World"</h2>
+          <h1>Women in the World Summit 2021</h1>
+        </div>
+        <p class="eventDetail">
+          W20 is the official G20 engagement group focused on gender equity. 
+          Its primary objective is to ensure that gender considerations are
+          mainstreamed into G20 discussions and translate into the G20 Leaders’ Declaration
+          as policies and commitments that foster gender equality and women’s economic empowerment.
+        </p>
+
+        <p class="furtherDetail"> Please contact us per Email for any further questions
+          about Women Summit 2021!</p>
+          <p class="emailInfo">womensummit2021@america.org</p>
```

Comment:
- [ ] This page has a horizontal overflow due to this section on mobile. Kindly fix this.

_See image below_

![image](https://user-images.githubusercontent.com/49621279/125054250-de870080-e05a-11eb-90ec-f302b112dc71.png)